pyscriptjs/src/pyodide.ts

```diff
@@ -78,6 +78,7 @@ export class PyodideInterpreter extends Interpreter {
             await this.loadPackage('micropip');
         }
         logger.info('pyodide loaded and initialized');
+        this.run('print("Python initialization complete")')
```

Comment:
I know we use this for tests, I wonder if maybe we could rely on the above logger instead, what do you think? 🤔

Reply:
In that case, it will not be present in `self.console.log.lines[0]`.
- It will be present in `self.console.info.lines`
- It will have a prefix i.e. the check will become `[pyscript/pyodide] pyodide loaded and initialized`
- It wouldn't be easily findable i.e. not a first OR last element. Thus, the check will be modified to become
`assert self.PY_COMPLETE in self.console.info.lines`

If that's okay, we can remove the `print` statement, and re-assign
`PY_COMPLETE` to `[pyscript/pyodide] pyodide loaded and initialized` instead.

Will that be okay?
CC: @JeffersGlass @FabioRosado

Reply:
We could perhaps turn it into console.log instead of `logger.info` that way it will show in log.lines. Although since this is just some of our testing logic we can also update the check to look into info.lines instead 😄

Reply:
I think we can address this in a separate PR if needed? Unless someone has strong opinions?